Adding missing release notes (#2789)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.42.0.Final/task-notification-emails.adoc

[id='task-notification-emails-742']

= Task Notification using Emails

> Where it was already possible to send task notification emails to specific users (by looking up the email associated with the provided user id), it now also possible to add additional email addresses directly.

Reviewer: ```suggestion
Earlier it was possible to send task notification emails to specific users (by looking up the email associated with the provided user id), now you can also add additional email addresses directly.
```
